Formula/cmdshelf.rb

```diff
@@ -0,0 +1,18 @@
+class Cmdshelf < Formula
+  desc "Better scripting life with cmdshelf"
+  homepage "https://github.com/toshi0383/cmdshelf"
+  url "https://github.com/toshi0383/cmdshelf/archive/2.0.0.tar.gz"
+  sha256 "3edb610cb512e147aa32e3e27e4c2d01a18ef738626304c42751c16a5b0fe309"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+    man.install Dir["docs/man/*"]
+    bash_completion.install "cmdshelf-completion.bash"
+  end
+
+  test do
+    assert_equal "2.0.0", shell_output("#{bin}/cmdshelf --version").chomp
```

Comment:
We'd like a `test` that exercises the __real__ functionality of the formula.

Reply:
Addressed in 6a2f99e9 🙏